src/setup.h

```diff
@@ -48,6 +49,9 @@
 #define VOLT_DIV_FACTOR 2
 #define BATTERY_LOW_LIMIT_VOLTAGE 3.3
 
+/** Exception priorities */
+#define PRIORITY_FACTOR 16
+
```

Comment:
This `PRIORITY_FACTOR` should probably be in the `.c` file, as it is not meant to be used outside the module?

Reply:
:+1: